formats/finder/frontend/examples/all_policies.json

```diff
@@ -0,0 +1,1109 @@
+{
+  "base_path": "/government/policies/all",
+  "title": "All policy content",
+  "description": "",
+  "format": "finder",
+  "need_ids": [],
+  "locale": "en",
+  "updated_at": "2015-07-28T13:31:24.082Z",
+  "public_updated_at": "2015-07-28T13:29:02.000+00:00",
+  "phase": "alpha",
```

Comment:
This an enormous example to include just to demonstrate the "phase" key. Could we use a shorter one?